Fix GetSpeakerCommand.cs Fix GetSpeakerAsRowCommand.cs

Tipson opened the pull request.

Tipson committed:
Fix GetSpeakerCommand.cs Fix GetSpeakerAsRowCommand.cs

Xpymb commented on src/Application/Common/Models/AgendaDto.cs:
```diff
@@ -9,7 +9,7 @@ namespace SynopsisV2.Application.Common.Models;
 {
     public Guid Id { get; set; }
-    public List<Speaker> Speakers { get; set; }
+    public List<Synopsis.Infrastructure.DbContext.Entities.Speaker> Speakers { get; set; }
```
Нужен SpeakerDto

Xpymb commented on src/Application/Common/Models/Speaker.cs:
```diff
@@ -7,7 +7,7 @@
 namespace SynopsisV2.Application.Common.Models;
 
-public class SpeakerDto : IMapFrom<Speaker>
+public class Speaker : IMapFrom<Synopsis.Infrastructure.DbContext.Entities.Speaker>
```
Переименуй обратно в SpeakerDto

Xpymb commented on src/Application/Partners/Commands/CreatePartner/CreatePartnerCommand.cs:
```diff
@@ -16,9 +16,9 @@ public record CreatePartnerCommand(
+    string Url) : IRequest<PartnerListDto>;
 
-public class CreatePartnerCommandHandle : IRequestHandler<CreatePartnerCommand, PartnerDto>
+public class CreatePartnerCommandHandle : IRequestHandler<CreatePartnerCommand, PartnerListDto>
```
Зачем возвращать список партнёров, если ты создаёшь всего одного партнёра?
Возвращай PartnerDto

Xpymb commented on src/Application/Partners/Commands/UpdatePartner/UpdatePartnerCommand.cs:
```diff
@@ -15,9 +15,9 @@ public record UpdatePartnerCommand(
+    string Url) : IRequest<PartnerListDto>;
 
-public class UpdatePartnerCommandHandler : IRequestHandler<UpdatePartnerCommand, PartnerDto>
+public class UpdatePartnerCommandHandler : IRequestHandler<UpdatePartnerCommand, PartnerListDto>
```
Тоже самое, обновляешь партнёра и зачем-то возвращаешь список партнёров

Xpymb commented on src/Application/Partners/Commands/UpdateVisibilityPartner/UpdateVisibilityPartnerCommand.cs:
```diff
@@ -10,9 +10,9 @@ namespace SynopsisV2.Application.Partners.Commands.UpdateVisibilityPartner;
+    bool IsShow) : IRequest<PartnerListDto>;
 
-public class UpdateVisibilityPartnerCommandHandler : IRequestHandler<UpdateVisibilityPartnerCommand, PartnerDto>
+public class UpdateVisibilityPartnerCommandHandler : IRequestHandler<UpdateVisibilityPartnerCommand, PartnerListDto>
```
И снова

Xpymb commented on src/Application/Partners/Queries/GetListPartner/GetListPartnerCommand.cs:
```diff
@@ -8,9 +8,9 @@
+public record GetListPartnerCommand(SynopsisVersionType VersionType) : IRequest<PartnerListDto>;
 
-public class GetListPartnerCommandHandler : IRequestHandler<GetListPartnerCommand, PartnerDto>
+public class GetListPartnerCommandHandler : IRequestHandler<GetListPartnerCommand, PartnerListDto>
```
Вот тут правильно

Xpymb commented on src/Application/Partners/Queries/GetPartner/GetPartnerCommand.cs:
```diff
@@ -7,9 +7,9 @@
+public record GetPartnerCommand(int Id) : IRequest<PartnerListDto>;
 
-public class GetPartnerCommandHandler : IRequestHandler<GetPartnerCommand, PartnerDto>
+public class GetPartnerCommandHandler : IRequestHandler<GetPartnerCommand, PartnerListDto>
```
И снова

Xpymb commented on src/Application/Speakers/Commands/CreateSpeaker/CreateSpeakerCommand.cs:
```diff
@@ -29,9 +30,9 @@ public record CreateSpeakerCommand(
+    int? PartnerId = null) : IRequest<Common.Models.Speaker>;
 
-public class CreateSpeakerCommandHandler : IRequestHandler<CreateSpeakerCommand, SpeakerDto>
+public class CreateSpeakerCommandHandler : IRequestHandler<CreateSpeakerCommand, Speaker>
```
SpeakerDto нужно возвращать

Xpymb commented on src/Application/Speakers/Commands/UpdateSpeaker/UpdateSpeakerCommand.cs:
```diff
@@ -32,9 +32,9 @@ public record UpdateSpeakerCommand(
+    int? PartnerId = null) : IRequest<Speaker>;
 
-public class UpdateSpeakerCommandHandler : IRequestHandler<UpdateSpeakerCommand, SpeakerDto>
+public class UpdateSpeakerCommandHandler : IRequestHandler<UpdateSpeakerCommand, Speaker>
```
SpeakerDto

Xpymb commented on src/Application/Speakers/Commands/UpdateVisibilitySpeaker/UpdateVisibilitySpeakerCommand.cs:
```diff
@@ -5,15 +5,15 @@
+    bool IsShow) : IRequest<Speaker>;
 
-public class UpdateVisibilitySpeakerCommandHandler : IRequestHandler<UpdateVisibilitySpeakerCommand, SpeakerDto>
+public class UpdateVisibilitySpeakerCommandHandler : IRequestHandler<UpdateVisibilitySpeakerCommand, Speaker>
```
SpeakerDto

Xpymb commented on src/Application/Speakers/Queries/GetListSpeakers/GetListSpeakerCommand.cs:
```diff
@@ -12,9 +12,9 @@
+    string Lang) : IRequest<Speaker>;
 
-public class GetListSpeakerCommandHandler : IRequestHandler<GetListSpeakerCommand, SpeakerDto>
+public class GetListSpeakerCommandHandler : IRequestHandler<GetListSpeakerCommand, Speaker>
```
ListSpeakersDto

Xpymb commented on src/Application/Speakers/Queries/GetSpeaker/GetSpeakerCommand.cs:
```diff
@@ -7,13 +7,14 @@
+    string Lang) : IRequest<Speaker>;
 
-public class GetSpeakerCommandHandler : IRequestHandler<GetSpeakerCommand, SpeakerDto>
+public class GetSpeakerCommandHandler : IRequestHandler<GetSpeakerCommand, Speaker>
```
SpeakerDto

Xpymb commented on src/Application/Speakers/Queries/GetSpeakerAsRow/GetSpeakerAsRowCommand.cs:
```diff
@@ -5,12 +5,13 @@
+public record GetAsRowSpeakerCommand(int Id) : IRequest<Speaker>;
 
-public class GetSpeakerAsRowCommandHandler : IRequestHandler<GetAsRowSpeakerCommand, SpeakerDto>
+public class GetSpeakerAsRowCommandHandler : IRequestHandler<GetAsRowSpeakerCommand, Speaker>
```
Тут всё правильно, нужно возвращать просто Speaker, полностью строку с бд

Xpymb commented on src/Infrastructure/Services/SendSmtp/SendSmtp.cs:
```diff
@@ -3,11 +3,11 @@
+public record SendSmtp(string Subject, string Body, string Recipient) : IRequest<Unit>;
 
-public class SendSmtpCommandHandler : IRequestHandler<SendSmtpCommand, Unit>
+public class SendSmtpCommandHandler : IRequestHandler<SendSmtp, Unit>
```
Удаляй команды для Smtp, они идут в SmtpService на слой Infrastructure

Tipson committed:
Fix AgendaDto.cs

Tipson committed:
Fix GetSpeakerAsRowCommand.cs

Tipson committed:
Fix name SpeakerListDto to SpeakerDto
Create SpeakerListDto.cs
Create PartnerListDto.cs
Fix GetListPartnerCommand.cs
Fix GetListSpeakerCommand.cs

Tipson committed:
Create All AgendaCommand

Xpymb commented on src/Application/Agendas/Queries/GetListAgendas/AgendaListDto.cs:
```diff
@@ -0,0 +1,6 @@
+
+namespace SynopsisV2.Application.Agendas.Queries.GetListAgendas;
+
+public record AgendaListDto(IEnumerable Agendas);
```
IEnumerable< AgendaDto > Agendas

Xpymb commented on src/Application/Feedbacks/Queries/SendFeedback/SendFeedbackCommand.cs:
```diff
@@ -1,45 +0,0 @@
-﻿using AutoMapper;
```
Верни

Xpymb commented on src/Application/Partners/Queries/GetListPartner/PartnerListDto.cs:
```diff
@@ -0,0 +1,6 @@
+
+namespace SynopsisV2.Application.Partners.Queries.GetListPartner;
+
+public record PartnerListDto(IEnumerable Partners);
```
IEnumerable< PartnerDto > Partners

Xpymb commented on src/Application/Speakers/Queries/GetListSpeakers/SpeakerListDto.cs:
```diff
@@ -0,0 +1,6 @@
+
+namespace SynopsisV2.Application.Speakers.Queries.GetListSpeakers;
+
+public record SpeakerListDto (IEnumerable Speakers);
```
IEnumerable<SpeakerDto> Speakers

Tipson committed:
Fix AgendaListDto.cs
Fix SpeakerListDto.cs
Fix PartnerListDto.cs
Re-Create SendFeedbackQuery.cs

Tipson committed:
Fix class name
Create GetGroupedPartnerQuery.cs
